packages/react-static-tweets/src/tweet.tsx

```diff
@@ -18,20 +18,24 @@ type TweetProps = {
 const Tweet = forwardRef<HTMLElement, TweetProps>(
   ({ id, ast, caption, className }: TweetProps, ref) => {
     const twitter = useTwitterContext()
-    const { data: tweetAst } = useSWR(id, twitter.swrOptions.fetcher, {
-      ...twitter.swrOptions,
-      fallbackData: ast || twitter.tweetAstMap[id]
-    })
+    const syncTweetAst = ast || twitter.tweetAstMap[id]
+    const { data: tweetAst } = useSWR(
+      syncTweetAst ? null : id,
+      twitter.swrOptions.fetcher,
+      {
+        ...twitter.swrOptions,
+        isPaused: () => !!syncTweetAst,
+        fallbackData: syncTweetAst
```

Comment:
Ideally if I just want to render the tweet and never update it, I shouldn't need the swr dep at all.

Reply:
Agreed.

I was originally trying to support both static and dynamic rendering with one component, but it'd be nice to have the guarantee that you'll never re-fetch / re-render the static tweets (unless you want to at the application level one step above this library).

If I remove the dynamic fetching support, then the biggest question becomes what to display when the tweet ast doesn't exist for a given tweet. Maybe just returning `null` for now and logging to the console is good enough?

Reply:
Maybe two entrypoints? a component that always expects `ast`, and a higher level component that can do its own fetching.